### ISSUE TITLE
Chart titles do not match the displayed info

### ISSUE THREAD

**Issue by stragu** [jaywink/the-federation.info#49]
"Active users last 30 days" gives the impression that the span of the chart is the last month, not the whole dataset of nearly three years. Same goes for "Active users last six months".

Instead of those two charts, it would be nice to see the actual evolution of the number of active users in the last 30 days (or is six months more relevant?), and in the last three years. I guess it is possible to show the last 30 days, as it looks like the resolution is one data point per day.

**Comment by jaywink** [comment]
I agree, it gives a really bad impression. I haven't been able to come up with a better title though. Basically the two sets of data there are representing the last 30 days or 6 month activities. Each day, the-federation.info will poll the pods and get the *current* values for these two stats. So each point in the graph is "users active within X days from this point backwards".

Unfortunately Diaspora pods are not giving out daily activity stats due to privacy reasons. It required a vote to even add the monthly stats - some people felt the statistics, the more "real time" it is could be used to track activity of a single user. So we compromised for 30 days as the most frequent stat. Daily stats would make representing in the graph much easier.

Any ideas on what to improve to make the graphs more interesting / reflect the stats collected?

**Comment by Flaburgan** [comment]
@stragu I'm currently working on a redesign of the website, you can access it at https://dev.the-federation.info if you have a better wording to propose please do. The data itself is not going to change.

**Comment by stragu** [comment]
Right, thanks @jaywink for the explanation and @Flaburgan for the heads up.

Maybe something like "active users in rolling 30-day period" and "active users in rolling 6-month period" would be more accurate?
